aptos-move/framework/aptos-framework/sources/Account.move

```diff
@@ -343,18 +343,95 @@ module AptosFramework::Account {
     }
 
     ///////////////////////////////////////////////////////////////////////////
-    /// Basic account creation method.
+    /// Basic account creation methods.
     ///////////////////////////////////////////////////////////////////////////
 
     public(script) fun create_account(auth_key: address) {
-        let (signer, _) = create_account_internal(auth_key);
+        let signer = create_account_internal(auth_key);
         Coin::register<TestCoin>(&signer);
     }
 
+    /// A resource account is used to manage resources independent of an account managed by a user.
```

Comment:
Speaking of resources vs modules, can we deploy modules with this account + SignerCap currently? My guess is no right since module bundle upload is a separate path currently.

Reply:
Per offline discussion, this can be left until later. @davidiw can you create an issue to track this so we don't forget?

Reply:
@areshand, this is something we should discuss tomorrow.